cli/slsa-verifier/main.go

```diff
@@ -24,8 +24,14 @@ var (
 	printProvenance bool
 )
 
+func experimentalEnabled() bool {
+	return os.Getenv("SLSA_VERIFIER_EXPERIMENTAL") == "1"
```

Comment:
can we just check if the env is set at all, rather than == 1?

something like
```
_, ok := os.LookupEnv("SLSA_VERIFIER_EXPERIMENTAL")
```

Reply:
sure

Reply:
done.

Reply:
Personally, I think we should look at it and verify it's not an empty string. It's pretty common practice to set env vars to empty strings to overwrite a previously set value rather than unsetting it.

Reply:
yeah, that's what happens when I used to do C/C++/Makefile builds. @asraa wdut?

Reply:
That's fair, the convention I was used to in the Sigstore tools was set/unset, and that's fine with me. I agree users sometimes can set a value to disable rather than unset it completely.

Reply:
Send the PR ill'll LGTM real quick.